.github/pull_request_template.md

```diff
@@ -1,29 +1,27 @@
 <!-- The following is intended to be helpful to you. Feel free to remove anything that is not. -->
 
-# References: 
-Jira: MNTOR-
-Figma: 
+# References:
 
+Jira: MNTOR-
+Figma:
 
 <!-- When adding a new feature: -->
 
 # Description
 
-
-
 # Screenshot (if applicable)
 
 Not applicable.
 
 # How to test
 
-
-
 # Checklist (Definition of Done)
+
 - [ ] Localization strings (if needed) have been added.
 - [ ] Commits in this PR are minimal and [have descriptive commit messages](https://chris.beams.io/posts/git-commit/).
 - [ ] I've added or updated the relevant sections in readme and/or code comments
 - [ ] I've added a unit test to test for potential regressions of this bug.
+- [ ] This PR requires a feature flag or experimentation, please notify the engineering team
```

Comment:
Hm, I think what will end up happening is that someone on the eng team will just set the right status in Jira. Should we suggest that here? ("If this PR implements a feature flag or experimentation, the Ship Behind Feature Flag status in Jira has been set")

Reply:
That's fine.

Reply:
```suggestion
- [ ] If this PR implements a feature flag or experimentation, the Ship Behind Feature Flag status in Jira has been set
```